niworkflows/utils/spaces.py

```diff
@@ -660,10 +660,14 @@ def __call__(self, parser, namespace, values, option_string=None):
             spaces.checkpoint()
         for val in values:
             val = val.rstrip(":")
-            # Should we support some sort of explicit "default" resolution?
-            # https://github.com/nipreps/niworkflows/pull/457#discussion_r375510227
```

Comment:
Can we keep this reference and add a reference to this PR as comments?

Reply:
sure